cmake/KokkosConfigCommon.cmake.in

```diff
@@ -306,3 +306,38 @@ FUNCTION(kokkos_compilation)
         ENDFOREACH()
     ENDIF()
 ENDFUNCTION()
+
+# this function is provided to easily set which files use the same programming language as Kokkos:
+#
+#       SOURCE      --> specific source files
+#
+FUNCTION(kokkos_set_source_language LANG_SOURCE)
```

Comment:
I would call it `kokkos_set_source_files[_language]` and now I start wondering if we should omit the suffix and reserve the right to set any property that might be relevant.

Reply:
I think we actually should reserve the right to set whatever we deem relevant for the file to be usable with the detected kokkos ... at least that is the idea of why they are introduced.